src/jit/importer.cpp

```diff
@@ -5727,10 +5733,10 @@ GenTreePtr Compiler::impImportStaticFieldAccess(CORINFO_RESOLVED_TOKEN* pResolve
             FieldSeqNode* fs = GetFieldSeqStore()->CreateSingleton(pResolvedToken->hField);
             op1              = gtNewOperNode(GT_ADD, type, op1,
                                 new (this, GT_CNS_INT) GenTreeIntCon(TYP_I_IMPL, pFieldInfo->offset, fs));
+            break;
```

Comment:
It was:
```
switch () {
  case : {

  }
  break;
} 
```

Now it is:
```
switch () {
  case : {
    break;
  }
}
```

I think both variants are equal. The second variant is used more often in this file. But I will return the old variant.